configurationdemo/resources/projects.xml

```diff
@@ -259,59 +245,65 @@ To launch this sample application on the Kaa C++ SDK:
 <ol>
 	<li>
 		Prepare your C++ SDK environment.
-		To do this, follow the instructions in <a href="http://kaaproject.github.io/kaa/docs/v0.10.0/Programming-guide/Using-Kaa-endpoint-SDKs/C++/" target="_blank">Using Kaa endpoint SDKs</a>.
+		To do this, follow the instructions in <a href="http://kaaproject.github.io/kaa/docs/latest/Programming-guide/Using-Kaa-endpoint-SDKs/C++/" target="_blank">Using Kaa C++ SDKs</a>.
 	</li>
 	<li>
 		Click the <b>Source</b> button.
 		A project source download window will open.
 		Click <b>Ok</b> and download the .tar.gz file.
 	</li>
 	<li>
-		Unpack the downloaded archive and run the following command from the directory containing the unpacked source files.
-		
-		<pre>
-			./build.sh deploy
-		</pre>
-		
+		Unpack the downloaded archive.
 	</li>
 	<li>
-		The program will ask you to enter the target platform for your build.
-		Press Enter to specify the posix platform (default) or enter the name of one of the <a href="http://kaaproject.github.io/kaa/docs/v0.10.0/Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/" target="_blank">supported platforms</a> and press Enter.
-		The C++ SDK and the demo application are now built and the demo application is launched.
+		Unpack the SDK archive.
+        <pre>
+            tar zxf libs/kaa/kaa* -C libs/kaa
+        </pre>
 	</li>
+	<li>
+		Run the following commands:
+		<p>
+			<b>Windows</b>
+			<pre>
+				mkdir build
+				cd build
+				cmake -G "Nmake Makefiles" ..
+				nmake
+			</pre>
+		</p>
+		<p>
+			<b>Linux, OS X (macOS)</b>
+			<pre>
+				mkdir build
+				cd build
+				cmake ..
+				make
+			</pre>
+            By default the project is generated for Posix target.
+            To generate and build project for other supported targets,
+            refer to the <a href="http://kaaproject.github.io/kaa/docs/latest/Programming-guide/Using-Kaa-endpoint-SDKs/" target="_blank">documentation pages</a>.
+		</p>
+	</li>
+    <li>
+        To run demo, execute:
+		<p>
+			<b>Windows</b>
+			<pre>
+                build/demo_client
```

Comment:
Also, please specify the file extention as well to prevent possible misunderstanding.